home/home5.2/src/ConfigLoader.java

```diff
@@ -0,0 +1,43 @@
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class ConfigLoader {
+    private static ConfigLoader instance;
+    private final Map<Character, String[]> config = new HashMap<>();
+
+    private ConfigLoader() {
+        loadConfig();
+    }
+
+    public static synchronized ConfigLoader getInstance() {
+        if (instance == null) {
+            instance = new ConfigLoader();
+        }
+        return instance;
+    }
+
+    private void loadConfig() {
```

Comment:
I hope we should return some status here - for example bool whether it was ok. Or we can throw an exception so the higher level of abstraction can handle it